unlock-app/src/components/interface/locks/CheckoutUrl/index.tsx

```diff
@@ -184,8 +187,14 @@ export const CheckoutUrlPage = () => {
       name: updated.name,
       config: updated.config as PaywallConfig,
     })
-    ToastHelper.success('Configuration updated.')
-    await refetchConfigList()
+
+    // Show toast message when configuration is changed after first render
```

Comment:
I am not sure I understand this part?

Reply:
When the checkout builder page loads, we set the last one the user updated as current configuration.
This will trigger the `onConfigSave` to auto-save, on the first-page load we don't need to show 'Configuration updated.' toast

Reply:
ok that makes sense